setup.cfg

```diff
@@ -19,10 +19,36 @@ description = Extension for colcon to extract reports from ASan and TSan output.
 long_description = file: README.rst
 keywords = colcon
 
+[options]
+install_requires =
+    colcon-core>=0.3.15
```

Comment:
A dependency on colcon-output introduced in #4. I don't have strong feelings about splitting this change up into that PR since they're all stacked.

Reply:
Good point. I don't mind keeping this here either, since we will merge the branches first before merging into master.

Reply:
> Did you mean for us to add `colcon-output>=0.2.3` in this file?

The library is required, since this PR adds dependency info for this package I would expect it to add all dependencies required.

Reply:
Added dependency.